packages/trader/src/Modules/Contract/Containers/contract.jsx

```diff
@@ -9,10 +9,20 @@ import { connect }       from 'Stores/connect';
 import ContractReplay    from './contract-replay.jsx';
 
 class Contract extends React.Component {
+    constructor () {
```

Comment:
can we use PureComponent here?

Reply:
Since we are using `connect` on this component, using `PureComponent` is not possible.